### PR TITLE
Move confirm order to checkout module

### PR DESCRIPTION
**Pull request by joeirimpan**
@prakashpp Please review [#131529715]

### CODE REVIEW

File: shop/cart/views.py [hunk 1]
```diff
@@ -80,29 +80,17 @@ def empty_cart():
...
-    if current_user.is_anonymous:
-        if not access_code:
-            # No access code provided, user is not authorized to
-            # access order page
-            abort(401)
-        if access_code != sale.guest_access_code:
-            # Invalid access code
-            abort(403)
-    else:
+    if not current_user.is_anonymous:
```
Comment (prakashpp) [inline shop/cart/views.py]: no need to check this, login required will take care of that

File: shop/cart/views.py [hunk 2]
```diff
@@ -80,29 +80,17 @@ def empty_cart():
...
         if sale.party.id != current_user.party.id:
```
Comment (prakashpp) [inline shop/cart/views.py]: just this needed

File: shop/cart/views.py (hunk 1 above)
Comment (joeirimpan) [inline shop/cart/views.py]: fixed

File: shop/cart/views.py (hunk 2 above)
Comment (joeirimpan) [inline shop/cart/views.py]: okay

**Pull request merged**